parla/array.py

```diff
@@ -5,6 +5,7 @@
 # FIXME: This load of numpy causes problems if numpy is multiloaded. So this breaks using VECs with parla tasks.
 #  Loading numpy locally works for some things, but not for the array._register_array_type call.
 import numpy as np
+import cupy as cp
```

Comment:
We should avoid this if at all possible. IIRC, importing cupy here prevents running the code at all without CUDA installed.